core/src/aztec/AZDecoder.cpp

```diff
@@ -321,24 +324,24 @@ DecoderResult Decode(const BitArray& bits)
 
 	StructuredAppendInfo sai = haveStructuredAppend ? ParseStructuredAppend(res.bytes) : StructuredAppendInfo();
 
-	// As converting character set ECIs ourselves and ignoring/skipping non-character ECIs, not using
-	// modifiers that indicate ECI protocol (ISO/IEC 24778:2008 Annex F Table F.1)
-	if (res.bytes.size() > 1 && res.bytes[0] == 29) {
```

Comment:
Any particular reason why you dropped that comment. Was there anything wrong with it?